### PR TITLE
Add project Evaluating Voting Design Tradeoffs for Retro Funding

### PR DESCRIPTION
**Pull request by idrees535**
This PR adds the project "Evaluating Voting Design Tradeoffs for Retro Funding" to the OSS Directory. The project is hosted at https://github.com/GovXS/Evaluating-Voting-Design-Tradeoffs-for-Retro-Funding and evaluates tradeoffs in voting mechanisms for retroactive funding.

Artifacts added:
https://github.com/GovXS/Evaluating-Voting-Design-Tradeoffs-for-Retro-Funding

### CODE REVIEW

File: data/projects/g/govxs.yaml [hunk 1]
```diff
@@ -0,0 +1,6 @@
+version: 2  # Ensure this is the latest version
```
Comment (ccerv1) [inline data/projects/g/govxs.yaml:1]: should be version: 7

File: data/projects/g/govxs.yaml
Comment (ccerv1) [inline data/projects/g/govxs.yaml:1]: The filename should match the `name` in the project file. So please rename to "evaluating-voting-design-tradeoffs-for-retro-funding-govxs.yaml" and move to the `e/` folder

**Review by ccerv1 — CHANGES_REQUESTED** [review]
Hi, thank you for your PR! Please fix the path/filename and the version.

File: data/projects/g/govxs.yaml (hunk 1 above)
Comment (idrees535) [reply data/projects/g/govxs.yaml]: Fixed it

File: data/projects/g/govxs.yaml
Comment (idrees535) [reply data/projects/g/govxs.yaml:1]: done

**Review by ccerv1 — APPROVED** [review]
lgtm!

**Pull request merged**